tools/wpt/testfiles.py

```diff
@@ -10,7 +10,7 @@
 here = os.path.dirname(__file__)
 wpt_root = os.path.abspath(os.path.join(here, os.pardir, os.pardir))
 
-logger = logging.getLogger()
+logger = logging.getLogger(wpt_root)
```

Comment:
Note that in testfiles (where the lack of root-logging setup was the issue) and check_stability are _both_ changed to wpt_root dir logging.